flare/otf.py

```diff
@@ -74,6 +74,12 @@ class OTF:
             single file name, or a list of several. Copied files will be
             prepended with the date and time with the format
             'Year.Month.Day:Hour:Minute:Second:'.
+        custom_module: A custom module that can be used in place of the DFT
```

Comment:
Just make sure to update this guy and to also scan for unit tests which might try to pass in 'dft_softwarename' -- as a side note, does your IDE have a `change name` function or similar which checks for usage within the same project? It can be quite nice to use!

Reply:
Update: didn't see that you had already done so below.  Sweet!